client/theme.js

```diff
@@ -74,11 +74,12 @@ const baseThemes = {
     primaryTextColor: grays.dark,
     inactiveTextColor: grays.middleDark,
     backgroundColor: grays.lighter,
+    accentColor: colors.p5jsPink,
 
     Button: {
       primary: {
         default: {
-          foreground: colors.black,
+          foreground: grays.mediumDark,
```

Comment:
This change will have implications on all button text colors throughout the app.  Is that what we want?  @shujuuu  Probably yes but I want to double check.

Reply:
yeah this is should be discussed